src/UI/Implementation/Component/Dropzone/Factory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Class Factory
+ *
+ * Default implementation for File Dropzone factory.
+ *
+ * @author  nmaerchy <nm@studer-raimann.ch>
+ * @date    22.05.17
+ * @version 0.0.1
+ *
+ * @package UI\Implementation\Component\Dropzone
```

Comment:
This represents the namespace in the comment and can be used in e.g. IDEs. A lot of classes in the UI-Service already have them.